src/app/app-shell/home/home.component.tns.scss

```diff
@@ -0,0 +1,48 @@
+$prototype-color: rgb(0, 198, 224);
+$prototype-color-bg: rgb(232, 246, 255) ;
+
+.main-container {
+  flex-direction: column-reverse;
+}
+
+.landscape.android740 :host .main-container {
+  flex-direction: row;
+}
+
+.menu-container {
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  height: 120 dp;
```

Comment:
Do we need to specify `dp`?

Reply:
I think we do, but I will investigate :)

Reply:
This example doesn't: https://docs.nativescript.org/ui/styling#supported-measurment-units
`.myLabel {
    font-size: 28;
    width: 200;
    height: 30;
}`

Reply:
I need to read about it before I beleive it :p